src/Stripe.net/Entities/Invoices/Invoice.cs

```diff
@@ -52,11 +52,13 @@ public class Invoice : StripeEntity<Invoice>, IHasId, IHasMetadata, IHasObject
         public bool AutoAdvance { get; set; }
 
         /// <summary>
+        /// Deprecated - use CollectionMethod
```

Comment:
Sorry I think I was unclear. https://stripe.com/docs/api/invoices/object#invoice_object-billing the docs explicitly say this now:
> This field has been renamed to collection_method and will be removed in a future API version.

So I think the docs in stripe-dotnet should say the exact same thing, and removing the old docs for the field entirely

In other places in the library we don't because the doc string hasn't been updated, but since we did here, it's just easier.

Reply:
got it.